src/components/Config/Config.tsx

```diff
@@ -8,9 +8,28 @@ import { capitalizeFirstLetter } from '../../utils';
 import CopyInput from '../CopyInput';
 import PageTitle from '../PageTitle';
 import ConnectionSettings from './ConnectionSettings';
+import { detect } from 'detect-browser';
 
 const { Link, Text } = Typography;
 
+function FirefoxAlert() {
```

Comment:
And actually, please move this to a separate `FirefoxAlert.tsx` file that will be in the same directory as this current file. This means you'll move the `detect` there as well obviously. Then you will import that component here.